net/librdkafka/Portfile

```diff
@@ -13,8 +13,8 @@ description         The Apache Kafka C/C++ library
 long_description    Full Apache Kafka protocol support, including producer and consumer
 homepage            https://github.com/edenhill/librdkafka
 
-checksums           sha256 79587c89a571b9db30635d9a4a55de624891ec34bcc1e5e5fd8736f05b17efa2 \
-                    rmd160 a624b87d5da3eb3bdb0182ebccf9376ee99f4773
+checksums           sha256 f0567b6abcdc597769995416d81f2e7ccf75ed8e69f2ed268b52ce120fd760c2 \
+                    rmd160 7c8ae28c27d5948e8b1e468b03492a9806af6e8b
```

Comment:
These days, we add a `size` field at the end of the checksums. `port lint` should have complained about that to you.

Reply:
Thanks, I just added the size! However I tried upgrading from MacPorts 2.5.3 to 2.5.4 and still do not see the lint error (below):
```
~/macports-ports/net/librdkafka$ port lint
--->  Verifying Portfile for librdkafka
--->  0 errors and 0 warnings found.
```